content/tutorial/02-sveltekit/01-concepts/04-server-side-rendering/README.md

```diff
@@ -2,4 +2,6 @@
 title: Server-side rendering
 ---
 
-TODO
+Server-side rendering (SSR) is the generation of the page contents on the server. It's what SvelteKit does by default when a user first visits your site.
```

Comment:
the 'first' could be interpreted as 'the first time ever', i.e. subsequent visits are controlled by a service worker with an app shell, PWA-style

```suggestion
Server-side rendering (SSR) is the generation of HTML on the server. It's what SvelteKit does by default for the initial page load when a user visits your site.
```